Tests/DependencyInjection/FixtureMonologExtensionTest.php

```diff
@@ -19,8 +20,13 @@
 
 abstract class FixtureMonologExtensionTest extends DependencyInjectionTest
 {
-    public function testLoadWithSeveralHandlers()
+    /** @group legacy */
+    public function testLegacyLoadWithSeveralHandlers()
     {
+        if (\class_exists(SwitchUserTokenProcessor::class)) {
```

Comment:
```suggestion
        if (class_exists(SwitchUserTokenProcessor::class)) {
```

same everywhere

Reply:
Shouldn't be this part of a code style check? Can we use fabbot here as well?